### PR TITLE
add pyuwsgi 2.0.21

### PR DESCRIPTION
**Pull request by mdtro**
Adding `pyuwsgi` 2.0.21 as prerequisite work to resolve https://github.com/getsentry/sentry/issues/43171.

### CODE REVIEW

File: packages.ini [hunk 1]
```diff
@@ -700,6 +700,9 @@ brew_requires =
 [pyuwsgi==2.0.20]
 python_versions = <3.11
 validate_skip_imports = uwsgidecorators
+[pyuwsgi==2.0.21]
+python_versions = <3.11
```
Comment (asottile-sentry) [inline packages.ini:704]: pyuwsgi has 3.11 wheels so you should remove this

**Comment by asottile-sentry** [comment]
@mdtro I don't think this actually helps / fixes the linked issue -- sounds like they have a conflict elsewhere (2.0.20 exists and works fine)

**Comment by mdtro** [comment]
> @mdtro I don't think this actually helps / fixes the linked issue -- sounds like they have a conflict elsewhere (2.0.20 exists and works fine)

Well, I guess I was going to upgrade the package and fix the issue at the same time. In our requirements-base.txt, we have a typo:

https://github.com/getsentry/sentry/blob/9d54da28406ca95baa3659b3d6e4a05d9ef5136b/requirements-base.txt#L73

The user is using `poetry` to install dependencies, so I assume it's just not able to handle the trailing `.0`.

**Comment by asottile-sentry** [comment]
> > @mdtro I don't think this actually helps / fixes the linked issue -- sounds like they have a conflict elsewhere (2.0.20 exists and works fine)
> 
> Well, I guess I was going to upgrade the package and fix the issue at the same time. In our requirements-base.txt, we have a typo:
> 
> https://github.com/getsentry/sentry/blob/9d54da28406ca95baa3659b3d6e4a05d9ef5136b/requirements-base.txt#L73
> 
> The user is using `poetry`, so I assume it's just not able to handle the trailing `.0` ?

2.0.20 and 2.0.20.0 and 2.0.20.0.0.0.0.0.0.0.0.0 are all the same version -- sounds like a bug in poetry and not our problem

**Comment by mdtro** [comment]
Opting to just fix the typo: https://github.com/getsentry/sentry/pull/43282

**Pull request closed without merging**